transaction.go

```diff
@@ -180,8 +180,15 @@ func (transaction *Transaction) DeleteCF(cf_handle *ColumnFamilyHandle, key []by
 // NewIterator returns an Iterator over the database that uses the
 // ReadOptions given.
 func (transaction *Transaction) NewIterator(opts *ReadOptions) *Iterator {
-	return NewNativeIterator(
-		unsafe.Pointer(C.rocksdb_transaction_create_iterator(transaction.c, opts.c)))
+	cIter := C.rocksdb_transaction_create_iterator(transaction.c, opts.c)
```

Comment:
Is there a reason for the extra variable? Isn't this the same as what was all rolled up together before?

Reply:
Not sure why I did that, I will switch them both back.

Reply:
Fixed.